### PR TITLE
Adds a slider to configure the object margin

### PR DESCRIPTION
**Pull request by Jummit**
Useful to space objects out a little more.

### CODE REVIEW

File: addons/zylann.scatter/tools/plugin.gd [hunk 1]
```diff
@@ -167,6 +167,7 @@ func _physics_process(delta):
 					var too_close = false
 					if len(_placed_instances) != 0:
 						var last_placed_transform = _placed_instances[-1].global_transform
+						if last_placed_transform.origin.distance_to(pos) < _pattern_margin + _palette.find_node("MarginSlider").value:
```
Comment (Zylann) [inline addons/zylann.scatter/tools/plugin.gd]: Don't find child nodes of a foreign scene in this way, it's like trying to access a private member. The node must be cached in `onready` of the palette and access to the value must be encapsulated with an accessor.

Comment (Jummit) [reply addons/zylann.scatter/tools/plugin.gd]: Done.

**Comment by Zylann** [comment]
I can merge this now, unless you prefer to wait for the previous PR

**Comment by Jummit** [comment]
> I can merge this now, unless you prefer to wait for the previous PR

Ready to merge.

**Comment by Zylann** [comment]
There are conflicts to resolve

**Comment by Jummit** [comment]
> There are conflicts to resolve

Resolved.

**Comment by Zylann** [comment]
Thanks!

**Pull request merged**